### PR TITLE
Obenkenobi/servler user 5.0

### PR DESCRIPTION
**Pull request by obenkenobi**
Added servlet user 5.0 instrumentation

### CODE REVIEW

File: instrumentation/servlet-user-5.0/src/main/java/jakarta/servlet/Filter.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ *  * Copyright 2020 New Relic Corporation. All rights reserved.
```
Comment (jasonjkeller) [inline instrumentation/servlet-user-5.0/src/main/java/jakarta/servlet/Filter.java:3]:
```suggestion
 *  * Copyright 2022 New Relic Corporation. All rights reserved.
```

File: instrumentation/servlet-user-5.0/src/main/java/jakarta/servlet/Servlet.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ *  * Copyright 2020 New Relic Corporation. All rights reserved.
```
Comment (jasonjkeller) [inline instrumentation/servlet-user-5.0/src/main/java/jakarta/servlet/Servlet.java:3]:
```suggestion
 *  * Copyright 2022 New Relic Corporation. All rights reserved.
```

File: instrumentation/servlet-user-5.0/build.gradle [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    implementation(project(":agent-bridge"))
+    implementation('jakarta.servlet:jakarta.servlet-api:5.0.0')
+}
+
+jar {
+    manifest { attributes 'Implementation-Title': 'com.newrelic.instrumentation.servlet-user-5.0', 'Enabled': 'false' }
+}
+
+verifyInstrumentation {
+    passesOnly 'jakarta.servlet:jakarta.servlet-api:[5.0.0,)'
```
Comment (jasonjkeller) [inline instrumentation/servlet-user-5.0/build.gradle:12]: I take it this instrumentation applies to servlet 6.0?

**Review by jasonjkeller — APPROVED** [review]

File: instrumentation/servlet-user-5.0/build.gradle (hunk 3 above)
Comment (obenkenobi) [reply instrumentation/servlet-user-5.0/build.gradle:12]: VerifyInstrumentation did pass for 6.0.

**Pull request merged**